src/test/java/study/SetCollectionTest.java

```diff
@@ -0,0 +1,47 @@
+package study;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+public class SetCollectionTest {
+	private Set<Integer> numbers;
+
+	@BeforeEach
+	void setUp() {
+		numbers = new HashSet<>();
+		numbers.add(1);
+		numbers.add(1);
+		numbers.add(2);
+		numbers.add(3);
+	}
+	
+	@Test
+	@DisplayName("step1.Test Set Collection size(size > 2, size <= 3)")
+	void size() {
+		assertThat(numbers).size().isGreaterThan(2)
+								  .isLessThanOrEqualTo(3);
+	}
```

Comment:
```suggestion
	void size() {
		assertThat(numbers.size()).isEqualTo(3);
	}
```
해당 테스트는 numbers 의 사이즈가 2보다 크고 3 보다 같거나 작은지에 대한 테스트 이네요 🤔 
`Set의 크기를 확인하는 학습테스트를 구현한다` 라는 요구사항과는 맞지 않는 것 같아요
isEqualTo() 를 활용해보면 어떨까요? 😄

Reply:
안녕하세요! 꼼꼼하게 봐주셔서 감사합니다~
앞으로 잘 부탁드립니다 🙌

조언해주신 내용 수정하여 push까지는 해두었는데용
해당 리소스가 merged된 상태는 완료되어서 반영하지 않아도 된다는 것으로 이해하면 될까요?

Reply:
다음 미션 리뷰 요청하실 때 수정하신 부분 반영해서 함께 리뷰 요청 주시면 될 거 같아요! 😄